scripts/app.ts

```diff
@@ -257,7 +267,5 @@ var actionProvider = {
     }
 };
 
-
-// Register context menu action provider
-VSS.register("blueprint.vsts-extension-split-work.vsts-extension-split-work-action", actionProvider);
```

Comment:
Can you help me understand this? We were registering an action and a dialog. It looks you've updated this to be "dialog" instead of "action", but in the vss-extension.json its still "action" for the contributed action menu.  I'm missing where this is loading the action.

Reply:
@davidcorrigan714 This is the only thing I'm concerned about. I thought this action was how we contribute into the drop down "..." menu. However, something may have changed since the old version of the sdk. If you have a chance to double check that would be great. Otherwise I'll merge this and confirm on my end. 

Thanks again for contributing! Its been very busy for me so I really appreciate the help. :)